jekyll/_cci2/persist-data.md

```diff
@@ -64,6 +64,14 @@ The information below describes how your network and storage usage is accumulati
 **Note:** Your overall **Network Transfer** amount is not representative of your billable usage. Only certain actions will result in network egress, which in turn results in billable usage. Details of these actions are described below.
 {: class="alert alert-info" }
 
+To view your network and storage usage follow these steps:
+
+1. Select **Plan** from the app sidebar.
+2. Select **Plan Usage**.
+3. Select the **Network** or **Storagek** tab depending on which you want to view.
```

Comment:
```suggestion
3. Select the **Network** or **Storage** tab depending on which you want to view.
```

Reply:
@rosieyohannan this is the only thing that really needs to be updated before merging, the rest is just suggestions :)